components/vision.py

```diff
@@ -64,6 +67,10 @@ def __init__(self) -> None:
         self.vision_data: Optional[VisionData] = None
 
         self.fuse_vision_observations = tunable(True)
+        # if we're sure we lost the target
+        self.lost_target = False
+        self.has_target = True
```

Comment:
Surely this should default to false? When we start we haven't seen the target yet.
```suggestion
        self.has_target = False
```